docs/guide/read_data_fragment.md

```diff
@@ -49,19 +49,6 @@ def preprocess(data, filter:FilterProtocol, detector:SpikeDetectionProtocol):
     return spiketrains
 ```
 
-## Read Data Fragmentally
-
-Instead of `data.load()`, try `data.load_fragments(num_fragments)` which internally splits the large datafile into
-number of fragmented data.
-
-```{code-cell} ipython3
-num_fragments = 100
-total_spikestamps = Spikestamps([])
-for data in tqdm(data.load_fragments(num_fragments=num_fragments), total=num_fragments):
-    spikestamp = preprocess(data, pre_filter, spike_detection)
-    total_spikestamps.extend(spikestamp)
-```
-
 ## Using Multiprocessing
```

Comment:
We can move this module into `tutorial/signal_processing.md` as well.

Reply:
Other than this part, we can probably remove this file entirely.
Also remove this file from `guide/index.rst` file.

Reply:
Okay, I think this is the last one before merging this PR. If you can remove the page `read_data_fragment` on `docs/guide/index.rst`, all CI will pass (probably).

If you want to build the documentation yourself, you can try [this](https://github.com/GazzolaLab/MiV-OS/blob/load_fragments_to_load/docs/README.md)